packages/create-sitecore-jss/src/initializers/react/args.ts

```diff
@@ -0,0 +1,4 @@
+﻿import { BaseArgs } from './../../common/args/base';
+import { ClientAppAnswer } from './../../common/prompts/base';
+
+export interface ReactArgs extends BaseArgs, Partial<Omit<ClientAppAnswer, 'fetchWith'>> {}
```

Comment:
Same as vue, we want `fetchWith` here. Extend `ClientAppArgs` instead of `BaseArgs`. Check one of the other initializers.